src/client/pages/OfferNew/utils.ts

```diff
@@ -544,3 +571,25 @@ export const getInsuranceTypesFromBundleVariant = (
 export const getQuoteIdsFromBundleVariant = (
   bundleVariant: QuoteBundleVariant,
 ) => bundleVariant.bundle.quotes.map((quote) => quote.id)
+
+export const getTypeOfContractFromBundleVariant = (
+  bundleVariant: QuoteBundleVariant,
+) => bundleVariant.bundle.quotes.map((quote) => quote.typeOfContract)
+
+function allBundleVariantsHaveSameInsuranceType(
```

Comment:
nit: consider using a function expression for consistency

Reply:
Yeah, thanks. This is what happens when I tell VSCode to create missing function declarations =)